src/SFML/Graphics/Text.cpp

```diff
@@ -317,8 +317,8 @@ Vector2f Text::findCharacterPos(std::size_t index) const
 
     // Precompute the variables needed by the algorithm
     bool  isBold          = m_style & Bold;
-    float whitespaceWidth = m_font->getGlyph(L' ', m_characterSize, isBold).advance + m_letterSpacing;
-    float lineSpacing     = m_font->getLineSpacing(m_characterSize)+ m_lineSpacing;
+    float whitespaceWidth = m_font->getGlyph(L' ', m_characterSize, isBold).advance * m_letterSpacingFactor;
```

Comment:
~~`float whitespaceWidth = m_font->getGlyph(L' ', m_characterSize, isBold).advance * m_letterSpacingFactor;`~~
`float whitespaceWidth = m_font->getGlyph(L' ', m_characterSize, isBold).advance;`
`float letterSpacing = ( whitespaceWidth / 3.0 ) * ( m_letterSpacingFactor - 1.0 );`
`whiteSpaceWidth += letterSpacing;`

Reply:
Clearance between characters is usualy 1/3 of the whitespace, so:
for m_letterSpacingFactor = 0.0 the spacing will be -1/3 of a whitespaceWidth (equiv. to -100 in Photoshop )
for m_letterSpacingFactor = 1.0 the spacing will be 1/3 of a whitespaceWidth (equiv. to  0 in Photoshop )
for m_letterSpacingFactor = 2.0 the spacing will be 1/3 of a whitespaceWidth (equiv. to +100 in Photoshop )
for m_letterSpacingFactor = 3.0 the spacing will be 2/3 of a whitespaceWidth (equiv. to +200 in Photoshop )